src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/AssetBundleVersion.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
nit: there's a new copyright header:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.